### ISSUE TITLE
Attribution section is including the copy button text

### ISSUE THREAD

**Issue by panchovm** [WordPress/openverse-frontend#1437]
## Description
Copying the text from **Credit the creator** section using the button includes the text of the button itself in the Chrome browser.

## Reproduction
1. Visit [this](https://search-staging.openverse.engineering/image/0fc8d459-6c59-4370-ae20-cc83c4a64e65) or any image.
2. Click the "copy" button for "Credit the creator" HTML tab.
3. Paste the content into a text editor and notice the "Copy text" added at the end

## Environment

 - Browser: Chrome 101.0.4951.64

## Additional context
Issue #1425 might be related.

## Resolution
- [ ] 🙋 I would be interested in resolving this bug.

**Comment by AetherUnbound** [comment]
Here's a video:

https://user-images.githubusercontent.com/10214785/168334089-aa6fcd29-ebad-4da4-8e8e-ac30e1336cd9.mp4

Note that on Pastebin it looks like the newline gets displayed as a proceeding space.

**Comment by sarayourfriend** [comment]
It's strange. I remembering being able to reproduce this but I can't any more. I only see the preceding space, no matter where I paste it (tried vim, pastebin, vscode, I tried just echoing it into my terminal).

It is weird that the preceding space is there though. I wonder if it just needs a `.trim()` call on the string?

**Comment by panchovm** [comment]
I can not watch or even download the video file.

**Comment by AetherUnbound** [comment]
It seems like Firefox does not like the video encoding - I am able to watch the video in Chrome though.

**Comment by sarayourfriend** [comment]
@AetherUnbound can you copy the character that's added on the first line? When I analyze what gets put into my clipboard, it's just a regular space character. I'm wondering if for you it's somehow something other than that.

Is anyone else able to reproduce this? @WordPress/openverse-maintainers

**Comment by AetherUnbound** [comment]
You know....I think this might just be gvim's pasting behavior 😂 😭 🤦🏼‍♀️ I _do_ remember seeing several newlines when I originally filed this, but that must have been fixed with the most recent frontend deployment. I think there's only the space at the beginning now, not any actual newlines. Should we leave this open to correct that? Or is that a non-issue?

**Comment by krysal** [comment]
I was able to reproduce this during the bug bash in Venice. I tried again and is kind of funny because I don't see new lines anymore but in Firefox it shows a space at the very beginning to me, and in Chrome it adds the text of the button after two new lines.

**Copying HTML from Chrome**

```
<p class="attribution">"<a target="_blank" rel="noopener noreferrer" href="https://www.flickr.com/photos/10511163@N00/520385031">Top Overview</a>" by <a target="_blank" rel="noopener noreferrer" href="https://www.flickr.com/photos/10511163@N00">Katharine Berry</a> is licensed under <a target="_blank" rel="noopener noreferrer" href="https://creativecommons.org/licenses/by-nc-sa/2.0/?ref=openverse">CC BY-NC-SA 2.0 <img src="https://mirrors.creativecommons.org/presskit/icons/cc.svg" style="height: 1em; margin-right: 0.125em; display: inline;"></img><img src="https://mirrors.creativecommons.org/presskit/icons/by.svg" style="height: 1em; margin-right: 0.125em; display: inline;"></img><img src="https://mirrors.creativecommons.org/presskit/icons/nc.svg" style="height: 1em; margin-right: 0.125em; display: inline;"></img><img src="https://mirrors.creativecommons.org/presskit/icons/sa.svg" style="height: 1em; margin-right: 0.125em; display: inline;"></img></a>. </p>


Copy text
```

**Copying Plain text from Chrome**
```
"Top Overview" by Katharine Berry is licensed under CC BY-NC-SA 2.0. To view a copy of this license, visit https://creativecommons.org/licenses/by-nc-sa/2.0/?ref=openverse.

Copy text
```

Note that this occurs in both production and staging.

**Comment by sarayourfriend** [comment]
I've update the issue to trim the whitespace. I'd say this is _very_ low priority. It might be trivial to fix but I could also see it being a ball of worms somehow.

The double space I'm not sure what to do about. How much do we care about that kind of thing?

**Comment by carlosreyesmunoz** [comment]
I am having a look to this issue.
I didn't manage to reproduce it by copying the content on a vsCode note, but with the text I also get the button text "copy text"

`
<p class="attribution">"<a target="_blank" rel="noopener noreferrer" href="https://www.flickr.com/photos/10511163@N00/520385031">Top Overview</a>" by <a target="_blank" rel="noopener noreferrer" href="https://www.flickr.com/photos/10511163@N00">Katharine Berry</a> is licensed under <a target="_blank" rel="noopener noreferrer" href="https://creativecommons.org/licenses/by-nc-sa/2.0/?ref=openverse">CC BY-NC-SA 2.0 <img src="https://mirrors.creativecommons.org/presskit/icons/cc.svg" style="height: 1em; margin-right: 0.125em; display: inline;"></img><img src="https://mirrors.creativecommons.org/presskit/icons/by.svg" style="height: 1em; margin-right: 0.125em; display: inline;"></img><img src="https://mirrors.creativecommons.org/presskit/icons/nc.svg" style="height: 1em; margin-right: 0.125em; display: inline;"></img><img src="https://mirrors.creativecommons.org/presskit/icons/sa.svg" style="height: 1em; margin-right: 0.125em; display: inline;"></img></a>. </p>


Copy text
`
just like that.

I would be interested on having a look at this

**Comment by sarayourfriend** [comment]
@carlosreyesmunoz That looks like the same result @krysal got too. Very strange!

I've assigned the issue to you :slightly_smiling_face: Feel free to ping if you need and assistance.

**Comment by krysal** [comment]
Fixed in #1458.

**Comment by krysal** [comment]
I created basically a duplicate of this issue in #1459 to specifically address the leading/trailing spaces and updated this one to reflect the problem with the button text included that @carlosreyesmunoz already fixed.

Thanks, Carlos! ✨
